components/prism-cil.js

```diff
@@ -0,0 +1,26 @@
+Prism.languages.cil = {
+	'displayTitle': "CIL",
+	
+	'comment': /\/\/[^\n]*(?=\r?\n)/,
+	
+	'string': {
+		pattern: /(["'])(?:\\(?:\r\n|[\s\S])|(?!\1)[^\\\r\n])*\1/,
+		greedy: true
+	},
+	
+	'class-name': {
+		lookbehind: true,
+		pattern: /([^a-zA-Z0-9])\.[a-z]+\b/
+	},
+	
+	// Actually an assembly reference
+	'variable': /\[\w+\]/,
+	
+	'boolean': /\b(?:true|false)\b/,
+	'number': /\b(0x)?[0-9a-fA-F]+\b/i,
+	'punctuation': /[{}[\];(),:]|IL_[0-9A-Za-z]+/,
+	
+	'keyword': /\b(iant|currency|syschar|void|bool|int8|int16|int32|int64|float32|float64|error|unsigned int8|unsigned int16|unsigned int32|unsigned int64|nativeType|decimal|date|bstr|lpstr|lpwstr|lptstr|objectref|iunknown|idispatch|struct|interface|int|unsigned int|nested struct|byvalstr|ansi bstr|tbstr|variant bool|lpstruct|static|public|private|family|final|specialname|virtual|abstract|assembly|default|cil managed|famandassem|famorassem|privatescope|hidebysig|newslot|rtspecialname|unmanagedexp|reqsecobj|pinvokeimpl|value|enum|interface|sealed|abstract|auto|sequential|explicit|ansi|unicode|autochar|import|serializable|nested public|nested private|nested family|nested assembly|nested famandassem|nested famorassem|beforefieldinit|specialname|rtspecialname|class|instance|string|extern|extends)\b/,
```

Comment:
A few things about this keyword list:

1. `initonly` is missing.
2. Please split things like `unsigned int32` into `unsigned` and `int32`. This will reduce the number of keywords.
3. Please sort the keywords alphabetically. This makes the list easier to maintain and you will notice the keywords which are included twice.